gensim/models/wrappers/ldamallet.py

```diff
@@ -358,12 +358,16 @@ def malletmodel2ldamodel(mallet_model):
     Args:
     ----
     mallet_model : Trained mallet model
+    gamma_threshold : To be used for LdaModel training.
```

Comment:
Misleading comment: there is no "training", the model is transferred from Mallet. These parameters only affect inference, model is unchanged.